backend/src/seeders/seeders.service.ts

```diff
@@ -19,8 +35,85 @@ export class SeederService {
 		fs.writeFileSync(this.shouldSeedFilePath, '');
 	}
 
+	async seedUsers() {
+		// passwords are visible
+		await this.userService.trySeed(seedData.users());
+	}
+
+	async seedGames() {
+		await this.gameService.trySeed(seedData.games());
+		const seededGames: Game[] = await this.gameService.findAll();
+		assert(seededGames.length === 3);
+
+		seededGames[0].score_player_one = 10;
+		seededGames[0].score_player_two = 5;
+		seededGames[0].is_active = false;
+
+		seededGames[1].score_player_one = 8;
+		seededGames[1].score_player_two = 3;
+		seededGames[1].is_active = false;
+
+		seededGames[2].score_player_one = 12;
+		seededGames[2].score_player_two = 7;
+		seededGames[2].is_active = false;
+		this.gameService.save(seededGames);
+	}
+
+	async seedChats() {
+		await this.chatService.trySeed(seedData.chats());
+	}
+
+	async seedUserChats() {
+		const allChats: Chat[] = await this.chatService.findAll();
+		assert(allChats.length === 3);
+
+		// get seeded users
+		const allUsers: User[] = await this.userService.findAll({
+			where: [
+				{ name: 'flamink' },
+				{ name: 'vaalboskat' },
+				{ name: 'renoster' },
+			],
+		});
+		assert(allUsers.length === 3);
+
+		// add users to chat, which create teh userCaht rel
+		allChats[0].users = allUsers;
+		allChats[1].users = allUsers.slice(0, 2);
+		allChats[2].users = allUsers.slice(1, 3);
+		await this.chatService.save(allChats);
+	}
+
+	async seedUserRelationships() {
+		const allUsers: User[] = await this.userService.findAll({
+			where: [
+				{ name: 'flamink' },
+				{ name: 'vaalboskat' },
+				{ name: 'renoster' },
+			],
+		});
+		const rel: CreateUserRelationshipDto[] = [
+			{
+				source_id: allUsers[0].id,
+				target_id: allUsers[1].id,
+				type: validRelationships.FRIEND,
+			},
+			{
+				source_id: allUsers[0].id,
+				target_id: allUsers[2].id,
+				type: validRelationships.BLOCKED,
+			},
+		];
+		await this.userRelService.save(rel);
+	}
```

Comment:
Depending on which branch will be merged first, this might fail (if new relationships are okay and pushed)
But I can take the seeding of that part after the branches are merged, so maybe just comment this out for now?